### DISCUSSION TITLE
Supplenza nella classe in cui insegno

### DISCUSSION THREAD

**Discussion by sempliciniluigi89** [iisgiua/giuaschool#370] [Idee e proposte]
La questione di non poter fare supplenza in una classe in cui si insegna rimane un mistero,è la prima volta che sento una cosa del genere ed è stranissimo ,neanche in lampschool era presente questo problema

**Comment by trinko** [comment]
E' stato pensato per evitare altri problemi... 
Attualmente potrebbe essere modificato, con una gestione un po' diversa della scelta delle classi. 
La segno come richiesta.
